bin/zeppelin-daemon.sh

```diff
@@ -159,6 +159,16 @@ function check_if_process_is_alive() {
   fi
 }
 
+function upstart() {
+
+  # upstart() allows zeppelin to be run and managed as a service
+  # for example, this could be called from an upstart script in /etc/init
+  # where the service manager starts and stops the process
+  initialize_default_directories
+
+  $ZEPPELIN_RUNNER $JAVA_OPTS -cp $ZEPPELIN_CLASSPATH_OVERRIDES:$CLASSPATH $ZEPPELIN_MAIN >> "${ZEPPELIN_OUTFILE}"
```

Comment:
This is purely for usage with a service manager such as upstart when found in ```/etc/init```
Its actually how I am running this in production on Ubuntu.  Upstart already detects a running process.  If you call
`sudo service zeppelin start`
then call again
`sudo service zeppelin start`
the service manager return a message that it is already started.


for example, here is a `zeppelin.conf`

```
description	"zeppelin"

start on (local-filesystems and net-device-up IFACE!=lo)
stop on shutdown

# Respawn the process on unexpected termination
respawn

# respawn the job up to 7 times within a 5 second period.
# If the job exceeds these values, it will be stopped and marked as failed.
respawn limit 7 5

# zeppelin installed in /usr/share/zeppelin in this example
chdir /usr/share/zeppelin
exec bin/zeppelin-daemon.sh upstart
```